app/scripts/types.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable */
+
```

Comment:
Created this file to consolidate shared types. Also nicer to do advanced TS things outside of JSDoc comments.